disco/extensions/upgrade_simulation/upgrades/common_functions.py

```diff
@@ -917,6 +921,10 @@ def get_all_line_info_instance(upper_limit=None, compute_loading=True, ignore_sw
     all_df = dss.utils.class_to_dataframe("line")
     if len(all_df) == 0:
         return pd.DataFrame()
+    flag = any(ele in ["Yes", "No"] for ele in all_df.enabled.unique())

```

Comment:
Do you want to make this a function since you are repeating it several times? If you want to make it a bit more robust in the future, you could make a case-insensitive check. If someone decided to change `true` to `Yes`, someone else could change `Yes` to `yes`.